### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by kcblaq**

### CODE REVIEW

**Review by pablo-clueless — COMMENTED** [review]
Please remove the commented code so I can see yours properly.

File: package.json [hunk 1]
```diff
@@ -27,8 +28,10 @@
     "react-otp-input": "^2.4.0",
     "react-redux": "^8.0.2",
     "react-router-dom": "^6.3.0",
-    "react-select": "^5.4.0",
-    "universal-cookie": "^4.0.4"
+    "universal-cookie": "^4.0.4",
+    "yup": "^0.32.11",
```
Comment (taiwoAkindele) [inline package.json:31]: Why are you importing this?

File: package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
     "@mui/styles": "^5.9.2",
     "@reduxjs/toolkit": "^1.8.4",
     "@types/react-redux": "^7.1.24",
+    "formik": "^2.2.9",
```
Comment (taiwoAkindele) [inline package.json:22]: Why?

**Review by taiwoAkindele — COMMENTED** [review]
Also I can't see your code on my browser don't know why

File: package.json (hunk 1 above)
Comment (kcblaq) [reply package.json:31]: I was'nt importing it, its a package someone else added on package.json.

I added just two packages, formik and yup, both for form handling and validation.

**Review by pablo-clueless — COMMENTED** [review]
Did you discuss the extra packages with Brain??

**Comment by PhilemonBrain** [comment]
@kcblaq please fix the conflicts

**Comment by PhilemonBrain** [comment]
@pablo-clueless check out thoose packages

**Pull request merged**